Libraries/Blob/File.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule File
+ * @flow
+ */
+'use strict';
+
+const Blob = require('Blob');
+
+/**
+ * The File interface provides information about files.
+ */
+class File extends Blob {
+  /**
+   * Name of the file.
+   */
+  get name(): string {
+    return this.data.name || '';
+  }
+
+  /*
+   * Last modified time of the file.
+   */
+  get lastModified(): number {
+    return this.data.lastModified || 0;
```

Comment:
Shouldn't we set `lastModified` to `createdAt` by default? Not sure how it is handled in web, but on my Mac if I create a file, it sets created/modified/last opened by default to creation time.

Reply:
@Kureev native side needs to pass this info when it's possible, but we can probably set it to current time from native if we cannot get the time.

Reply:
Based on what I see as an outcome of `new File([""], "");`, `lastModified` is set to `Date.now()` by default

Reply:
@Kureev, yes, we do it here https://github.com/facebook/react-native/pull/11573/files#diff-2bf13477704f772255c64e0beb652478R63

Reply:
Ditto, this should just be an invariant that `lastModified` is a number (if it's just to make Flow happy).